tsMuxer/srtStreamReader.cpp

```diff
@@ -192,7 +192,7 @@ uint8_t* SRTStreamReader::renderNextMessage(uint32_t& renderedLen)
         m_state = PARSE_TIME;
         bool isNUmber = true;
         {
-            for (auto c : m_sourceText.front())
+            for (auto& c : m_sourceText.front())
```

Comment:
Just out of curiosity, what's the reason behind this? There doesn't seem to be any benefit to accessing this container via reference vs. via value.

Reply:
Just to be consistent with all other ranged-based for loops: there is no need to copy the value to another variable here.

Reply:
I was going to say "but it's potentially more costlier to use a reference here", but a quick peek at [godbolt](https://gcc.godbolt.org/z/sG84qT) proved me wrong : the code is literally the same in both cases.